### PR TITLE
Adds roach riding + fixes roach taming!

### PR DESCRIPTION
**Pull request by Kmc2000**
NOW WITH 100% LESS SHITCODE!!!!
https://streamable.com/v9l6d

To tame a roach:
(optional) for better results, do this as ironhammer or someone with high vigilance
Grow ambrosia vulgaris, or something sugary, drug laden, or vodka filled.
Grow at least 2, if not more!
Make wired rod
Click wired rod with your plant
Go find roach in maint
Hold your plant in ACTIVE HAND
Once you see it hesitate, BUCKLE INTO IT
If successful, the roach will look stoned
To move your new roach, simply hold your plant on a stick out and click where you want the roach to go. WATCH OUT! If you click too much, the roach might eat the plant on a stick.

NOTA BENE:
THE ROACH WHEELCHAIR IS JUST FOR MEMES, it is NOT used in the code.

### CODE REVIEW

File: code/datums/mob_stats.dm [hunk 1]
```diff
@@ -112,9 +112,6 @@
 				qdel(SM)
 			. += SM.value
 
-
```
Comment (Kmc2000) [inline code/datums/mob_stats.dm]: I am the ghost of random whitespace.

**Comment by seekin4u** [comment]
Could you do something with roach movement?
Why human's move is faster and edgy? Do roaches have another movement code or what?

**Comment by drexample** [comment]
Is this being worked on?

**Comment by drexample** [comment]
I'll take that as a no.

**Pull request closed without merging**